docs/experimental-features/security-posture-management.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[security-posture-management]]
+== Cloud security posture management
+
+NOTE: This feature requires Elastic Stack version 8.3.0 or later.
+
+Cloud security posture management (CSPM) and Kubernetes security posture management (KSPM) continuously monitor and compare your cloud and Kubernetes infrastructure against security best practices to help you identify and remediate misconfigurations.
+
+[discrete]
+=== Enable security posture management
+
+The steps to enable this feature differ between Elastic cloud and self-hosted deployments.
+
+[[enable-kspm-on-cloud]]
+[discrete]
+==== Enable KSPM for cloud deployments
+
+First, enable the KSPM flag for your deployment:
```

Comment:
Echoing @nastasha-solomon's comment about making _all_ of these steps numbered, and sub-numbered/alpha. It's hard to follow with actions in different formats, when they're all just steps to follow. Something like:

1. Enable the KSPM flag:
   a. From Kibana main menu, click Manage this...
   b. Under My deployment...
   [ . . . ]
   f. Click Back then save...

2. Navigate back to Kibana...

3. Follow the prompts...